packages/react-component-library/src/components/Dropdown/DropdownLabel.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+import { Visibility, VisibilityOff } from '../../icons'
```

Comment:
If we can delay merging this i'll hopefully have a fix for the tree shaking issue today.

Reply:
I'd rather merge it and then retrofit the improvements then it wont get held up if the other ticket does.